code/modules/atmospherics/machinery/pipes/layermanifold.dm

```diff
@@ -37,7 +38,7 @@
 /obj/machinery/atmospherics/pipe/layer_manifold/proc/get_all_connected_nodes()
 	return front_nodes + back_nodes + nodes
 
-/obj/machinery/atmospherics/pipe/layer_manifold/update_icon()	//HEAVILY WIP FOR UPDATE ICONS!!
+/obj/machinery/atmospherics/pipe/layer_manifold/update_icon()
```

Comment:
Is it no longer WIP? I realllly hope it's no longer WIP, cause I doubt it'll be ever done if it is.

Reply:
idk if I remember correctly I was the last one to rewrite icon logic for the layer manifold and I don't remember writing this so I guess this is an oversight from back then